api/index.md

```diff
@@ -5,13 +5,570 @@ _navPath: toc.html
 _navRel: ../toc.html
 ---
 
-# .NET Core API Reference
+# .NET Core API Reference 
 
-These documents cover the API reference for .NET Core. You can use the search function 
-on the left-hand side to search for an API or use the Table of Contents on the left to navigate.
+The .NET Core API Reference documents the types and type members available in .NET Core. You can navigate to the documentation for a particular type (a class, structure, delegate, enum, or interface) or type member (a constructor, method, property, or event) in any of three ways:
```

Comment:
If you're enumerating all the possible members, why do you leave out fields? Sure, fields are rare in public APIs, but [they do exist](http://dotnet.github.io/api/System.String.html#fields).